extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -522,8 +532,8 @@ async function setupDeleteExcludeTest(proj: Project): Promise<[ProjectEntry, Pro
 	await proj.addScriptItem('UpperFolder/LowerFolder/someOtherScript.sql', 'Also not a real script');
 
 	const projTreeRoot = new ProjectRootTreeItem(proj);
-
-	testContext.apiWrapper.setup(x => x.showWarningMessageOptions(TypeMoq.It.isAny(), TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(() => Promise.resolve(constants.yesString));
+	sinon.stub(vscode.window, 'showWarningMessage').returns(<any>Promise.resolve(constants.yesString));
+	// testContext.apiWrapper.setup(x => x.showWarningMessageOptions(TypeMoq.It.isAny(), TypeMoq.It.isAny(), TypeMoq.It.isAny())).returns(() => Promise.resolve(constants.yesString));
```

Comment:
this should be removed instead of commented out?

Reply:
Oops, thanks for being diligent =D